sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3086,6 +3086,19 @@ object GpuOverrides extends Logging {
          |For instance decimal strings not longer than 18 characters / hexadecimal strings
          |not longer than 15 characters disregarding the sign cannot cause an overflow.
          """.stripMargin.replaceAll("\n", " ")),
+    expr[FormatNumber](
+      "Formats numeric column x to a format like '#,###,###.##', rounded to d decimal places" +
+      " with HALF_EVEN round mode, and returns the result as a string column.\n" + 
+      "If d is 0, the result has no decimal point or fractional part. If d is less than 0, " + 
+      "the result will be null.",
+      ExprChecks.binaryProject(TypeSig.STRING, TypeSig.STRING,
+        ("x", TypeSig.cpuNumeric, TypeSig.cpuNumeric),
```

Comment:
Can we change this to gpuNumeric for the GPU version? They are the same so it should not change anything, but to be defensive it would be good to keep them separate in case spark adds a new number format.

Reply:
done